index.html

```diff
@@ -1469,10 +1469,17 @@ <h2 id="funding">Parcels funding and support</h2>
 <!-- Footer -->
 <footer class="py-5 bg-dark">
   <div class="container">
-    <p class="m-0 text-center text-white">
-      Copyright &copy; OceanParcels project, 2018<br>
-      <small>Based on the <a href ="https://blackrockdigital.github.io/startbootstrap-modern-business/">Modern Business</a> theme, distributed by <a href="https://startbootstrap.com/template-overviews/modern-business/">Start Bootstrap</a> under an MIT license</small>
-    </p>
+    <div class="row">
+    <ul id="footerul">
+      <li id="footerli"><a id="footera" href="accessibility.html">Accessibility</a></li>
+<!--      <li><a href="#news">Cookie policy</a></li>-->
+      <li id="footerli"><a id="footera" href="utrechtteam.html">Utrecht Team</a></li>
+    </ul>
+  </div>
+  <p class="m-0 text-center text-white">
+    Copyright &copy; OceanParcels project, 2018<br>
```

Comment:
Should we not update this to 2020?
```suggestion
    Copyright &copy; OceanParcels project, 2020<br>
```